shuffle-plugin/src/main/scala/com/nvidia/spark/rapids/shuffle/ucx/UCX.scala

```diff
@@ -129,14 +153,49 @@ class UCX(executorId: Int, usingWakeupFeature: Boolean = true) extends AutoClose
 
       var workerParams = new UcpWorkerParams()
 
-      if (usingWakeupFeature) {
+      if (rapidsConf.shuffleUcxUseWakeup) {
         workerParams = workerParams
           .requestWakeupTagSend()
           .requestWakeupTagRecv()
       }
 
       worker = context.newWorker(workerParams)
       logInfo(s"UCX Worker created")
+      if (rapidsConf.shuffleUcxUseSockaddr) {
+        // For now backward endpoints are not used and just add them as negative executorId.
```

Comment:
I assume the backwards endpoints are required for the listener callbacks to be executed? Or could we not create the backwards endpoints now and still get the error callbacks?

Reply:
If they are required as is, even though we are not using them to send messages, mind adding this in the comment?